frontend/src/types.ts

```diff
@@ -696,3 +712,9 @@ export type ContextResourceData<T> = {
   error?: Error;
   refresh: () => void;
 };
+
+export type BreadcrumbItemType = {
+  label: string;
+  link?: string;
+  isActive?: boolean;
+};
```

Comment:
Nit. Force one or the other prop.

```suggestion
export type BreadcrumbItemType = {
  label: string;
} & EitherNotBoth<{ link: string }, { isActive: boolean }>;
```